api/v1alpha4/azurecluster_conversion.go

```diff
@@ -40,6 +40,38 @@ func (src *AzureCluster) ConvertTo(dstRaw conversion.Hub) error { // nolint
 	// Restore list of virtual network peerings
 	dst.Spec.NetworkSpec.Vnet.Peerings = restored.Spec.NetworkSpec.Vnet.Peerings
 
+	if restored.Spec.NetworkSpec.NodeOutboundLB != nil {
+		for i, _ := range restored.Spec.NetworkSpec.NodeOutboundLB.FrontendIPs {
+			dst.Spec.NetworkSpec.NodeOutboundLB.FrontendIPs[i].PublicIP = restored.Spec.NetworkSpec.NodeOutboundLB.FrontendIPs[i].PublicIP
```

Comment:
shouldn't we only set publicIP.outboundIP?

Reply:
missed it, will add only outboundIP here